src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -135,6 +139,8 @@ public boolean isPassable(IBlockAccess worldIn, BlockPos pos)
         return false;
     }
 
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings ("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)